content/sensu-go/5.15/guides/use-federation.md

```diff
@@ -0,0 +1,313 @@
+---
+title: "Multi-cluster visibility with federation"
+linkTitle: "Reaching Multi-cluster Visibility"
+description: "In this guide, you'll learn how federate Sensu clusters and use a single web UI to access resources across those disparate clusters."
```

Comment:
I'm not sure I understand what "a single web UI" means here. It sounds like it might mean you can manage resources across all federated clusters from only one screen when you log into the web UI, but is that correct?

Also, can you only access the resources? Or can you manage the resources too?

Reply:
Based on the description in the early paragraphs of the doc, I suggest changing the description to:

"Learn how to federate Sensu clusters. With Sensu's federation feature, you can access resources across multiple clusters via the web UI and mirror your changes in one cluster to follower clusters."

Reply:
> I'm not sure I understand what "a single web UI" means here. It sounds like it might mean you can manage resources across all federated clusters from only one screen when you log into the web UI, but is that correct?

Yes I think your read is correct. Upon completing this guide, I can log into the web UI running on the `gateway` cluster and interact with resources in the `gateway`, `alpha` and `beta` clusters. Without federation, I'd need to have a distinct web UI browser session for each cluster I want to interact with.

> Also, can you only access the resources? Or can you manage the resources too?

The interactions that are allowed are a function of my RBAC permissions. Assuming one has followed this guide, an authenticated session with `cluster-admin` role can both view and manage resources, whereas the `federation-viewer` user will be allowed only to view resources.

Reply:
Got it. I will make a suggestion.

Reply:
I think I may have addressed this now. Please lmk what you think.

Reply:
Sounds great! 👍